examples/example-vite-react-sql/src/hooks/useQuery.ts

```
@@ -0,0 +1,39 @@
+import { useQuery as useTanStackQuery } from "@tanstack/react-query";
+
+const fetchSchema = async (query: string, formatFn: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)